azblob/url_append_blob.go

```diff
@@ -66,7 +67,7 @@ func (ab AppendBlobURL) Create(ctx context.Context, h BlobHTTPHeaders, metadata
 		ifModifiedSince, ifUnmodifiedSince, ifMatch, ifNoneMatch,
 		nil, // Blob tags
```

Comment:
should this comment remain ?

Reply:
ifTags ( x-ms-if-tags conditional header to specify blob only if a specified condition is met) specify a SQL where clause on blob tags to operate only on blobs with a matching value. So wherever in function signature ifTags is mentioned, it facilitates the user to add a where clause. Since the primary task was to add set tags/get tags/find blob by tags, I did not add these changes. It can be rolled out separately. For reference, I looked at PR of other SDK ([.net](https://github.com/Azure/azure-sdk-for-net/pull/11060), [Python](https://github.com/Azure/azure-sdk-for-python/pull/11418))

Reply:
thanks for confirming. The comment however is same for this param and for param changed few lines below and might be causing confusion.

Reply:
Ohh I understand. I can do the changes but I do not wish to change it because 
1. These changes can be delivered separately with the current work. 
2. These changes were made when we added support for 2019-12-12 service version. If I do the changes, it also changes the git history of that line. 
What do you suggest?

Reply:
I suggest to leave codebase in better shape and make these changes.

Reply:
Done